azure-monitor-opentelemetry-distro/setup.cfg

```diff
@@ -32,7 +32,10 @@ zip_safe = False
 include_package_data = True
 install_requires =
     azure-monitor-opentelemetry-exporter == 1.0.0b7
-    opentelemetry-instrumentation == 0.32b0
+    opentelemetry-instrumentation == 0.33b0
```

Comment:
Should we add some tests that make sure these load properly? Similar to how we test out the exporters.